com.github.muriloventuroso.givemelyrics.json

```diff
@@ -17,9 +17,25 @@
         "--metadata=X-DConf=migrate-path=/com/github/muriloventuroso/givemelyrics/"
     ],
     "modules": [
+        {
+            "name": "libsoup",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                   "type":"git",
+                   "url":"https://gitlab.gnome.org/GNOME/libsoup.git",
+                   "tag":"2.74.1"
+                }
+            ]
+        },
         {
             "name": "givemelyrics",
             "buildsystem": "meson",
+            "post-install": [
+                "install -Dm644 /app/share/icons/hicolor/24x24@2/apps/com.github.muriloventuroso.givemelyrics.svg /app/share/icons/hicolor/scalable/apps/com.github.muriloventuroso.givemelyrics.svg",
+                "mv /app/share/icons/hicolor/24x24/actions/* /app/share/icons/hicolor/scalable/actions/",
+                "rm -r /app/share/icons/hicolor/[0-9]*"
```

Comment:
why do you remove these?

Reply:
For the error non-png-icon-in-hicolor-size-folder. There are .svg in nxn folders.
https://buildbot.flathub.org/#/builders/6/builds/143748